.github/actions/setup-ci-env-vars/action.yml

```diff
@@ -0,0 +1,34 @@
+name: Set CI Environment Variables
+description: Set CI Environment Variables
+runs:
+   using: 'composite'
+   steps:
+      - name: Get Action Job Info
+        uses: octokit/request-action@v2.x
+        id: get_action_job_data
+        with:
+           route: GET /repos/${{ github.repository }}/actions/runs/${{github.run_id}}/jobs
+
+      - name: Get job url and start date
+        id: get_job_url_and_date
+        run: |
+           JOB_DATA_JSON=$(echo $job_data_json | jq '.jobs | .[]? | select(.name == "${{ env.GITHUB_JOB_NAME }}")')
+           echo "job-url=$(echo $JOB_DATA_JSON | jq '.html_url')" >> $GITHUB_OUTPUT
+           echo "job-start-time=$(echo $JOB_DATA_JSON | jq '.started_at')" >> $GITHUB_OUTPUT
+        env:
+           job_data_json: ${{ steps.get_action_job_data.outputs.data }}
+        shell: bash
+
+      - name: Set Environment Variables
+        run: |
+           echo "QUEUED_AT=$(date --date="${{ steps.get_job_url_and_date.outputs.job-start-time }}" +%s)" >> $GITHUB_ENV
```

Comment:
This isn't used. How are we getting the queued at time for the action run?

Reply:
Now this is used in https://github.com/iFixit/react-commerce/commit/170799c0ddb8fb77f36cc1866acbd6589625472e.